blob-module/ios/BlobStream.swift

```diff
@@ -0,0 +1,29 @@
+
+class BlobStream {
+    var data: [Data]
+    var currentIndex = 0
+    
+    init(data: [Data]) {
+        self.data = data
+    }
+    
+    func getReader() -> BlobStreamReader {
+        return BlobStreamReader(data: data)
+    }
+}
+
+class BlobStreamReader {
+    var data: [Data]
+    var currentIndex = 0
+    
+    init(data: [Data]) {
+        self.data = data
+    }
+    
+    func readAll() async throws -> Data {
```

Comment:
I don't think this is a method of the `Reader` interface https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader